src/styles/blocks.scss/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 166px;
+  max-height: 16px;
+  margin: 0;
+  padding-top: 17px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../../images/star.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+  &__star:nth-child(-n + 4) {
+    background-image: url(../../images/star-active.svg);
+  }
+
+  &__reviews {
+    padding-left: 33px;
+    font-family: Roboto, Helvetica, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    color: #060b35;
+  }
+}
```

Comment:
copy all logic from the stars task, we must have the ability to change the nr of active stars by changing the modifier